ipywidgets/static/widgets/js/layout.js

```diff
@@ -36,18 +36,15 @@ define([
                 'align_items',
                 'align_self',
                 'bottom',
+                'border',
```

Comment:
Just because the box model accounts for it in its calculations, doesn't mean it should be used to conduct layout.  Instead padding or margin should be used.